yoyodyne/models/base.py

```diff
@@ -235,6 +235,28 @@ def validation_epoch_end(self, validation_step_outputs: Dict) -> Dict:
             self.log(metric, value, prog_bar=True)
         return metrics
 
+    def predict_step(
+        self,
+        batch: batches.PaddedBatch,
+        batch_idx: int,
+    ) -> torch.Tensor:
+        """Runs one predict step.
+
+        This is called by the PL Trainer.
+
+        Args:
+            batch (batches.PaddedBatch).
+            batch_idx (int).
+
+        Returns:
+            torch.Tensor: indices of the argmax at each timestep.
+        """
+        self.eval()
+        predictions = self(batch)
+        # Default tensor: B x seq_len x vocab_size.
```

Comment:
What does "Default tensor" mean? Is this just saying the next step returns something of shape B x seq_len x vocab_size? If so then use Adam's notation:

`# -> B x seq_len x vocab_size.`